specification/compliance.md

```diff
@@ -0,0 +1,23 @@
+# FOCUS Compliance
+
+The FOCUS specification is designed to assist FinOps practitioners in performing common FinOps capabilities by utilizing a standardized set of instructions, regardless of the billing data generator. It does so by establishing compliance criteria that account for provider-level variances in the cost and usage datasets providers produce. This section outlines the two primary types of cost datasets that FinOps practitioners currently work with and explains how FOCUS can enhance the interoperability between these different datasets.
```

Comment:
> This section outlines the two primary types of cost datasets that FinOps practitioners currently work with and explains how FOCUS can enhance the interoperability between these different datasets.

What "two types of cost datasets" is this referring to? This isn't clear.

Reply:
Removed this section